app/models/reason/PaymentPenaltyReasonEnum.scala

```diff
@@ -34,6 +43,15 @@ object PaymentPenaltyReasonEnum   extends Enumeration {
         case "VAT_NOT_PAID_WITHIN_15_DAYS" => JsSuccess(VAT_NOT_PAID_WITHIN_15_DAYS)
         case "VAT_NOT_PAID_WITHIN_30_DAYS" => JsSuccess(VAT_NOT_PAID_WITHIN_30_DAYS)
         case "VAT_NOT_PAID_AFTER_30_DAYS"  => JsSuccess(VAT_NOT_PAID_AFTER_30_DAYS)
```

Comment:
Scala checker complains 'Cyclomatic complexity of 14 exceeds max of 10. ' against these lines of case . Are we sure we can ignore it.

Reply:
The team has agreed to ignore complexity warning for this specific situation